packages/components/src/entities/SampleAliquotsGridPanel.tsx

```diff
@@ -216,7 +134,8 @@ interface SampleAliquotsGridPanelProps extends Omit<Props, 'queryModelId'> {
     omittedColumns?: string[];
     rootLsid?: string;
     sampleId: string | number;
-    sampleLsid: string; // if sample is an aliquot, use the aliquot's root to find subaliquots
+    sampleLsid: string;
+    // if sample is an aliquot, use the aliquot's root to find subaliquots
```

Comment:
I assume this was done by the linter, but the comment should be moved above `sampleLsid` right?

Reply:
yep. thanks